pkg/controller/import-controller.go

```diff
@@ -250,9 +250,14 @@ func (ic *ImportController) createImporterPod(pvc *v1.PersistentVolumeClaim, pvc
 		return err
 	}
 
+	podResourceRequirements, err := GetDefaultPodResourceRequirements(ic.cdiClient)
```

Comment:
Same as clone controller, I think it makes sense to look up the pod requirements in the function that builds the spec.

Reply:
DONE